### PR TITLE
Eco places

### PR DESCRIPTION
**Pull request by VadymMakitra**

### CODE REVIEW

File: src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts [hunk 1]
```diff
@@ -151,15 +186,21 @@ export class CreateEditEventsComponent implements OnInit {
...
-
-    if (this.checkdates && this.titleForm.valid && this.contentValid) {
+    if (this.editMode) {
+      sendEventDto = {
+        ...sendEventDto,
+        imagesTodelete: this.imagesToDelete
+      };
+    }
+    const test = true;
+    if (this.checkdates && this.eventFormGroup.valid && tagsArr.length && test) {
```
Comment (hnativlyubomyr) [inline src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts]: Why you use test here? It is always true

File: src/app/main/component/events/components/images-container/images-container.component.ts [hunk 2]
```diff
@@ -70,8 +95,18 @@ export class ImagesContainerComponent implements OnInit {
     this.images.splice(i, 1);
     this.imgArray.splice(i, 1);
     this.imgArrayOutput.emit(this.imgArray);
-    this.images.push({ src: null, label: this.dragAndDropLabel, isLabel: false });
+    if (this.editMode) {
+      this.deleteImagesOutput.emit(this.imagesTodelete);
+    }
+    let allowLabel = false;
+    if (i === this.imagesCount - 1) {
```
Comment (hnativlyubomyr) [inline src/app/main/component/events/components/images-container/images-container.component.ts]: remove if
let allowLabel = i === this.imagesCount - 1;

File: src/app/main/component/events/services/events.service.ts [hunk 3]
```diff
@@ -23,6 +23,10 @@ export class EventsService implements OnDestroy {
     return this.http.post<any>(`${this.backEnd}events/create`, formData);
   }
 
+  public editEvent(formData: FormData): Observable<any> {
```
Comment (hnativlyubomyr) [inline src/app/main/component/events/services/events.service.ts]: Why any? boolean

Comment (hnativlyubomyr) [reply src/app/main/component/events/services/events.service.ts:26]: Observable<boolean>

File: src/app/main/component/places/components/add-place/add-place.component.scss [hunk 4]
```diff
@@ -31,36 +23,27 @@
...
-}
-
 .input {
   width: 100%;
   padding: 8px;
   border-radius: 4px;
   line-height: 20px;
   font-size: 14px;
   border: 1px solid var(--ubs-quintynary-light-grey);
+  background-color: white;
```
Comment (hnativlyubomyr) [inline src/app/main/component/places/components/add-place/add-place.component.scss]: use --primary-white

File: src/app/main/component/places/components/time-picker-pop-up/time-picker-popup.component.ts [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+  save(): void {
+    this.timeOfWork.emit(
+      this.selectTheWorkDay.map((day): WorkingTime => {
+        return { dayOfWeek: day.dayToSend, timeTo: day.timeTo, timeFrom: day.timeFrom, isSelected: day.isSelected };
+      })
+    );
+  }
+
+  setTime(index) {
```
Comment (hnativlyubomyr) [inline src/app/main/component/places/components/time-picker-pop-up/time-picker-popup.component.ts]: setTime(index: number): void

File: src/app/store/reducers/tariff.reducer.ts [hunk 6]
```diff
@@ -27,7 +27,13 @@ export const tariffReducer = createReducer(
       stateLocation.locationsDto.forEach((locationDto, locationDtoIndex) => {
         action.editedLocations.forEach((actionEditedLocation) => {
           if (locationDto.locationId === actionEditedLocation.locationId) {
-            stateLocations[stateLocationIndex].locationsDto[locationDtoIndex].locationTranslationDtoList = actionEditedLocation.location;
+            stateLocations[stateLocationIndex].locationsDto[locationDtoIndex].locationTranslationDtoList.forEach((location) => {
+              if (location.languageCode === 'ua') {
```
Comment (hnativlyubomyr) [inline src/app/store/reducers/tariff.reducer.ts]: use ternary operator here

File: src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts (hunk 1 above)
Comment (VadymMakitra) [reply src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts]: @hnativlyubomyr Hi, I got some issues with Github, this change is not related to this pr.

File: src/app/main/component/events/components/images-container/images-container.component.ts (hunk 2 above)
Comment (VadymMakitra) [reply src/app/main/component/events/components/images-container/images-container.component.ts]: @hnativlyubomyr Hi, I got some issues with Github, this change is not related to this pr.

File: src/app/main/component/events/services/events.service.ts (hunk 3 above)
Comment (VadymMakitra) [reply src/app/main/component/events/services/events.service.ts]: Hi, I got some issues with Github, this change is not related to this pr.

File: src/app/main/component/places/components/add-place/add-place.component.scss (hunk 4 above)
Comment (VadymMakitra) [reply src/app/main/component/places/components/add-place/add-place.component.scss]: Done

File: src/app/main/component/places/components/time-picker-pop-up/time-picker-popup.component.ts (hunk 5 above)
Comment (VadymMakitra) [reply src/app/main/component/places/components/time-picker-pop-up/time-picker-popup.component.ts]: Done

File: src/app/store/reducers/tariff.reducer.ts (hunk 6 above)
Comment (VadymMakitra) [reply src/app/store/reducers/tariff.reducer.ts]: Hi, I got some issues with Github, this change is not related to this pr.

**Review by NewArtMask — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by Myh45 — APPROVED** [review]

**Review by melnychenkoas — APPROVED** [review]

**Review by krayevska — APPROVED** [review]

**Review by Mukola1983 — APPROVED** [review]

**Pull request merged**